lib/theme_check/checks/missing_enable_comment.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module ThemeCheck
+  class MissingEnableComment < LiquidCheck
+    severity :error
+
+    # Don't allow this check to be disabled with a comment,
+    # as we need to be able to check for disabled checks.
+    always_enable!
```

Comment:
I think changing to `disableable false` (or `can_disable`) would make it clearer what this does.